app/views/directives/add-config-to-application.html

```diff
@@ -31,22 +31,42 @@
         <legend>Add {{ctrl.apiObject.kind | humanizeKind}} as:</legend>
         <div class="form-group">
           <div class="radio">
-            <div class="add-choice">
-              <label>
-                <input type="radio" ng-model="ctrl.addType" value="env" ng-disabled="ctrl.disableInputs">
-                Environment variables
-              </label>
+            <label class="add-choice" for="env">
+              <input id="env" type="radio" ng-model="ctrl.addType" value="env" ng-disabled="ctrl.disableInputs">
+              Environment variables
+            </label>
+            <div class="choice-options">
+              <div class="form-horizontal">
+                <span class="control-label">Prefix</span>
```

Comment:
Should we be using `class="form-group"` and are the `choice-options` styles still necessary if we do?